src/documentation/markdown.rs

```diff
@@ -71,15 +70,46 @@ impl<'a> PlainOverlay<'a> {
                                 // @todo validate as if it was another document entity
                             }
                         }
+                        Tag::Link(link_type, _url, title) => {
+                            // @todo check links
+                            // for now, only dealing with some links types
+                            match link_type {
+                                LinkType::Inline => {}
+                                LinkType::Autolink | LinkType::Email  => skip_link_text = true,
```

Comment:
I actually checked and I could not reproduce those types extra types. Either does not recognize them correct because they are not yet supported or I am getting the wrong examples to test (not sure, there are not many in the docs).

https://docs.rs/pulldown-cmark/0.7.2/pulldown_cmark/enum.LinkType.html

Reply:
Can you elaborate? I am not sure I can follow.